js/data.js

```diff
@@ -88,7 +88,7 @@ class Data extends AdaptCollection {
     if (Adapt.get('_isStarted')) {
       Adapt.set('_isStarted', false);
     }
-    this.loadCourseData(language, previousLanguage);
+    await this.loadCourseData(language, previousLanguage);
```

Comment:
This isn't necessary. 

`onLanguageChange` is triggered by a backbone event, which doesn't wait for promises to complete. So there is nothing waiting for `onLanguageChange` to complete and therefore no need to await the final call to `loadCourseData`. You can let this function-call hang in the breeze.

Equally, leaving this in isn't a problem. It just doesn't do anything.